addons/preview-project-description/addon.json

```
@@ -0,0 +1,21 @@
+{
+  "name": "Preview Project Descriptions",
+  "description": "Allows you to preview the what will show up in the final notes and credits.",
```

Maybe something like...

```suggestion
  "name": "Preview project instructions and notes",
  "description": "In your own projects, adds a toggle [somewhere TODO] that lets you preview the project instructions and notes as seen by other users.",
```